flask_fixtures/__init__.py

```diff
@@ -33,8 +33,13 @@
 except ImportError:
     import json
 
-__version__ = '0.3.7'
+__version__ = '0.3.9'
 
+# storing tables loaded in the fixtures
+_fixture_tables = []
+
+# storing fixtures loaded from filesystem
+_fixture_dict = {}
```

Comment:
These are globals, so I'm assuming that these should have the scope of the entire process (test run) - which would be true if you're persisting fixtures across multiple tests

Reply:
If they should have the scope of one `TestCase`, they should be class scoped

Reply:
Yeah, I will fix this.

`_fixture_dict` has the scope of entire test run, (it stores the fixture jsons which are same for all tests)

`_fixture_tables` have the scope of a class only where we store the tables loaded in a particular test.